packages/devextreme/js/__internal/ui/tree_view/tree_view.search.ts

```diff
@@ -190,7 +190,9 @@ class TreeViewSearch extends TreeViewBase {
   _itemContainer(isSearchMode?: boolean, selectAllEnabled?: boolean): dxElementWrapper {
     const isSelectAllEnabled = selectAllEnabled ?? this._selectAllEnabled();
 
-    if (isSelectAllEnabled) {
+    const { items } = this.option();
```

Comment:
```suggestion
    const { items = [] } = this.option();
```

Reply:
After this you can remove optional changing from the condition

Reply:
i did, but why is it a preferrable way?
if items were used more than once (or if actually needed items) -- then sure
but in this case it's just `?` against ` = [] ` and optional chaining is shorter and still readable and understandable